go/l1client/txhandler/handler.go

```diff
@@ -0,0 +1,168 @@
+package txhandler
+
+import (
+	"fmt"
+	"math/big"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/obscuronet/obscuro-playground/go/buildhelper/buildconstants"
+	"github.com/obscuronet/obscuro-playground/go/log"
+	"github.com/obscuronet/obscuro-playground/go/obscurocommon"
+	"github.com/obscuronet/obscuro-playground/go/obscuronode/nodecommon"
+)
+
+const methodBytesLen = 4
+
+type TxHandler interface {
+	PackTx(tx *obscurocommon.L1TxData, from common.Address, nonce uint64) (types.TxData, error)
+	UnPackTx(tx *types.Transaction) *obscurocommon.L1TxData
+}
+
+type EthTxHandler struct {
+	contractABI abi.ABI
+}
+
+func NewEthTxHandler() TxHandler {
+	contractABI, err := abi.JSON(strings.NewReader(buildconstants.ContractAbi))
```

Comment:
the abi file should be moved here as well (from buildconstants)

Reply:
Done 👍